docs/source/invoking-with-neonova.md

```diff
@@ -0,0 +1,28 @@
+# Invoking smart contracts with NeoNova
+
+After writing, compiling and deploying your smart contract locally, you can invoke it using [NeoNova](https://www.neonova.space/).
+
+First, connect to your local network by clicking on the top right button next to "Select Wallet", then "Custom" and 
+input your local address with the rpc-port and magic number (they should be inside the .neo-express file), and click on
+"Save".
+
+![Setting Local Network](./images/InvokeNeoNova_1Network.gif)
+
+If you want to change or add data to the blockchain, you'll also need to connect a wallet to sign your transactions,
+click on the "Select Wallet" button, choose one of the platforms and then click on "Connect" (you need to have the app
+you chose on your device).
+
+![Adding Wallet](./images/InvokeNeoNova_2Wallet.gif)
+
+Then, select the "Read" or "Write" type on the menu depending on what you want to do, input the script hash of the smart 
```

Comment:
> depending on what you want to do

It would be interesting to explain the differences and that you can "test" your "write" invocations, etc.